tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -512,9 +513,14 @@ struct RTFFile {
 /// A zero UUID stored at the end of the TFile record
 struct RTFUUID {
    RUInt16BE fVersionClass{1};
-   unsigned char fUUID[16] = {0};
+   unsigned char fUUID[16];
```

Comment:
That seems undersized ...

Reply:
Clarified by the discussion with @silverweed, AFAICT